connectors/rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/RmqSourceReplicator.java

```diff
@@ -89,26 +90,27 @@ private synchronized void startMQAdminTools() throws MQClientException {
     }
 
     @Override
-    public String verifyAndSetConfig(KeyValue config) {
+    public void validate(KeyValue config) {
 
         // Check the need key.
         for (String requestKey : ConfigDefine.REQUEST_CONFIG) {
             if (!config.containsKey(requestKey)) {
-                return "Request config key: " + requestKey;
+                return;
```

Comment:
Missing configuration should throw an exception instead of returning directly. Returning directly indicates that the verification has passed

Reply:
> Missing configuration should throw an exception instead of returning directly. Returning directly indicates that the verification has passed

done